src/SUFST/inc/can_unpack.h

```diff
@@ -3,9 +3,22 @@
 #include "tx_api.h"
 #include "telemetry_protocol.h"
 #include "rtcan.h"
+#include <stdint.h>
 
 #define CAN_PUBLISHER_RX_QUEUE_SIZE 10 //TODO: add config.h storing such values to avoid redefinition.
 
+typedef struct {
+
+  TX_TIMER bps_timer; /* Timer definition */
+  TX_MUTEX stats_mutex; /* Mutex definition */
+  int32_t rx_can_count; /* Received can frames */
+  int32_t rx_can_bps; /* Received bits per second */
+  int32_t rxbytes;
+  int32_t tx_pdu_count; /* Transmitted pdu frames */
+  int32_t tx_pdu_bps; /* Transmitted bits per second */
+  int32_t txbytes;
+} unpack_performance_t;
```

Comment:
Maybe we should call this `unpack_stats_t`, I know I made the issue about "performance" but on reflection calling it stats is clearer.